api.raml

```diff
@@ -193,6 +186,30 @@ schemas:
             "required": false,
             "type": "string",
             "enum": ["yes", "no"]
+          },
+          "password": {
```

Comment:
nevermind, I was not picking up on the fact that the schema was referring to what's coming in on the POST/PUT.  Corrected in https://github.com/silinternational/idp-id-broker/commit/c922f30c8971af1e3d0bd518d58a6e4e9be7817a